pkg/ebpf/c/tracee.bpf.c

```diff
@@ -747,7 +747,8 @@ enum
 
 struct modules_map {
     __uint(type, BPF_MAP_TYPE_HASH);
-    __uint(max_entries, MAX_NUM_MODULES);
+    __uint(max_entries, 2 * MAX_NUM_MODULES); // Do not decrease as it won't be big enough to hold
```

Comment:
consider using another constant for this to be more clear, e.g MAX_MODULES_MAP_ENTRIES